doc/source/serve/advanced.rst

```diff
@@ -185,6 +185,24 @@ The shard key can either be specified via the X-SERVE-SHARD-KEY HTTP header or `
   handle = serve.get_handle("api_endpoint")
   handler.options(shard_key=session_id).remote(args)
 
+Composing Multiple Models
+=========================
+In many cases, you need to use multiple models to perform a prediction. It 
+could that you are connecting a preprocess model with a inference model; or
+using multiple models to perform stacking or ensembles. Ray Serve has built in
+support for composing models together.
+
+To define a higher-level composed model, you can define a Ray Serve backend and
+connect to an endpoint normally. With the following differences:
+
+- You can retrieve a "handle" to your physical models and call it with Python 
+  arguments via ``serve.get_handle("endpoint-name")``
+- You should make your backend implementation async python function with ``async def``.
```

Comment:
```suggestion
In order to make calls to the underlying models, you can retrieve a "handle" to their endpoints and call them with Python using ``serve.get_handle("endpoint")``. Additionally, in order to avoid blocking the composed model while waiting for requests, it should be made an async function using ``async def``.
```